cmd/testrunner.go

```diff
@@ -23,32 +23,44 @@ import (
 	_ "github.com/elastic/elastic-package/internal/testrunner/runners" // register all test runners
 )
 
+func init() {
+	cobraext.CommandInfos[testCmd] = cobraext.CommandInfo{
```

Comment:
Frankly speaking, I'm not in favor of overusing `init` as it's more like static code - you never know the order of execution.

WDYT about a bit different approach:
- `cobraext` will contain a structure that extends `cobra.Command` with context (global, package) - let's say `cobraext.Command`
- leave single `init` function in `root.go` that will collect and return all `cobraext.Command`s
- no need to expose CommandInfos, which are dedicated only for README generation.

I'm happy to discuss this idea.

Reply:
I like it! I think it's a better design. I'll take a stab at it and update this PR. Thanks!

Reply:
I took a stab at this in c0176d7. Let me know what you think!